spec/models/custom_attribute_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe CustomAttribute do
+  it "return correct field type" do
+    string_custom_attribute = FactoryGirl.create(:custom_attribute, :name => "foo", :value => "string", :resource_type => 'ExtManagementSystem')
+    time_custom_attribute = FactoryGirl.create(:custom_attribute, :name => "bar", :value => DateTime.current, :resource_type => 'ExtManagementSystem')
+    int_custom_attribute = FactoryGirl.create(:custom_attribute, :name => "foobar", :value => 5, :resource_type => 'ExtManagementSystem')
+
+    expect(string_custom_attribute.field_type).to eq(:string)
+
+    expect(time_custom_attribute.field_type).to eq(:datetime)
+
+    expect(int_custom_attribute.field_type).to eq(:fixnum)
```

Comment:
Is `:fixnum` a valid field type? It doesn't appear to be in Postgres: https://www.postgresql.org/docs/8.0/static/datatype.html#DATATYPE-NUMERIC